frontend/tests/sql-editor/nav.spec.ts

```diff
@@ -0,0 +1,163 @@
+  ).toBeTruthy()
+})
+
+test('SQL editor opens and can click on new query 2', async ({ page }) => {
+  await page.goto(editorUrl)
+
+  const gettingStartedLoaded = page.getByText(/Getting started/)
+  await expect(gettingStartedLoaded).toBeVisible()
+
+  // wait for API calls to finish
+  const entityDefinitions = await page.waitForResponse((r) => {
+    return r.url().includes('/query?key=entity-definitions') && r.request().method() === 'POST'
+  })
+  expect(entityDefinitions.ok()).toBeTruthy()
+
+  // create first snippet
+  await page.click('"New query"', { delay: 20 })
+
+  const createFirstSnippetResp = await page.waitForResponse((r) => {
+    return r.url().includes('/content') && r.request().method() === 'POST'
+  })
+  await expect(createFirstSnippetResp.ok()).toBeTruthy()
+
+  const snippetOneUrl = await page.url()
+
+  // Wait for Monaco Editor to be ready
+  const monacoEditor = await page.locator('.monaco-editor').first()
+
+  // Get the Monaco Editor instance
+  await monacoEditor.waitFor({ state: 'visible' })
+
+  // Click on the Monaco Editor
+  await monacoEditor.click({ delay: 20 })
+
+  // Focus the Monaco Editor
+  //   await monacoEditor?.focus()
+
+  // Emulate typing using low-level key events
+  await page.keyboard.press(`${cmdKey}+A`) // Select all existing text
+  await page.keyboard.press('Backspace') // Delete the selected text
+  await page.keyboard.type('select * from public.city') // Type the desired text
+
+  await monacoEditor.focus()
+  await page.keyboard.press(`${cmdKey}+Enter`, { delay: 100 })
+
+  const firstSqlQueryResp = await page.waitForResponse((r) => {
+    return r.url().includes('/query') && r.request().method() === 'POST'
+  })
+
+  expect(firstSqlQueryResp.ok()).toBeTruthy()
+  await expect(page.getByText(/Abha/)).toBeVisible()
+  await expect(page.getByText(/Abu Dhabi/)).toBeVisible()
+  await expect(page.getByText(/Acua/)).toBeVisible()
+
+  // create second snippet
+  await await page.click('"New query"')
+
+  const createSecondSnippetResp = await page.waitForResponse((r) => {
+    return r.url().includes('/content') && r.request().method() === 'POST'
+  })
+  await expect(createSecondSnippetResp.ok()).toBeTruthy()
+
+  const snippetTwoUrl = await page.url()
+
+  // Click on the Monaco Editor
+  await monacoEditor.click()
+  // Emulate typing using low-level key events
+  await page.keyboard.press(`${cmdKey}+A`) // Select all existing text
+  await page.keyboard.press('Backspace') // Delete the selected text
+  await page.keyboard.type('select * from public.actor') // Type the desired text
+
+  // switch between snippets
+  await page.click(`a[href="${snippetOneUrl.replace('https://app.supabase.green', '')}"]`)
+  await page.click(`a[href="${snippetTwoUrl.replace('https://app.supabase.green', '')}"]`)
+  await page.click(`a[href="${snippetOneUrl.replace('https://app.supabase.green', '')}"]`)
+  await page.click(`a[href="${snippetTwoUrl.replace('https://app.supabase.green', '')}"]`)
```

Comment:
don't use base path as string: tests for prod or vercel build would fail, use env vars instead:

```suggestion
  await page.click(`a[href="${snippetOneUrl.replace(dashboardUrl, '')}"]`)
  await page.click(`a[href="${snippetTwoUrl.replace(dashboardUrl, '')}"]`)
  await page.click(`a[href="${snippetOneUrl.replace(dashboardUrl, '')}"]`)
  await page.click(`a[href="${snippetTwoUrl.replace(dashboardUrl, '')}"]`)
```